packages/image-utils/src/Image.ts

```diff
@@ -73,13 +79,9 @@ async function resizeAsync(imageOptions: ImageOptions): Promise<Buffer> {
 
     if (imageOptions.borderRadius) {
       const mask = Buffer.from(
-        `<svg><rect x="0" y="0" width="${width}" height="${height}" 
-        rx="${imageOptions.borderRadius}" ry="${imageOptions.borderRadius}" 
-        fill="${
-          backgroundColor && backgroundColor !== 'transparent' ? backgroundColor : 'none'
-        }" /></svg>`
+        `<svg><circle cx="${imageOptions.borderRadius}" cy="${imageOptions.borderRadius}" r="${imageOptions.borderRadius}" /></svg>`
```

Comment:
why do we want to change this to a circle? with a rect we can support different rounded corners rather than just a circle

Reply:
because it wasn't cropping the image at all

Reply:
I think that's because of a backgroundColor change I made that broke this, though, not because of using a `rect`

Reply:
^ problem is that background color for `roundBackgroundLayer` is always `transparent` now, so `fill` here is always `none`, so it wouldn't crop. We could default to like `#ffffff` here or something instead